.vscode/settings.json

```diff
@@ -17,10 +17,9 @@
     "logs/**": true
   },
   "python.testing.pytestArgs": [
-    "."
+    "tests"
```

Comment:
Sorry, this is reverting a change which snuck in before - I think I'm going to have trouble devving without changing this stuff by mistake?